config-model/src/main/java/com/yahoo/vespa/model/admin/metricsproxy/MetricsProxyContainer.java

```diff
@@ -13,13 +13,34 @@
  */
 public class MetricsProxyContainer extends Container {
 
-    public MetricsProxyContainer(AbstractConfigProducer parent) {
-        super(parent, "" + 0, 0);
+    public MetricsProxyContainer(AbstractConfigProducer parent, int index) {
+        super(parent, "" + index, index);
     }
 
     @Override
     protected ContainerServiceType myServiceType() {
         return METRICS_PROXY_CONTAINER;
     }
 
+    @Override
+    public int getWantedPort() {
+        return 19092; // TODO: current metrics-proxy uses 19091 as rpc port, will now get 19093.
```

Comment:
What about clients that will try to use the old port? How/when should they switch to the new port number?

Reply:
The final version will most likely use the current port.

Reply:
👍